### ISSUE TITLE
Labels Not Properly Associated with Inputs

### ISSUE THREAD

**Issue by Shitanshukumar607** [EduHaven/EduHaven#434]
Most of the form inputs and textareas are missing id attributes, and their labels are not using htmlFor, which affects accessibility and screen reader support
